### PR TITLE
Fix/condo/doma 1896/upeate icons

### PR DESCRIPTION
**Pull request by VKislov**
Сhanged side menu icons according to the layout.
Before: 
image

After: 
image

Created from PR: https://github.com/open-condo-software/condo/pull/1629
Recreated branch because the branch was not deployed for review

### CODE REVIEW

**Comment by VKislov** [comment]
I left `BarMiniAppsIcon` uncompressed because after processing the fill does not work correctly.

**Review by sitozzz — APPROVED** [review]

**Review by akarjakin — APPROVED** [review]

**Review by SavelevMatthew — CHANGES_REQUESTED** [review]
As far as I remember we have monochromatic icons, yours are two-coloured. You should pass design review before merging this PR. And why are you passing viewbox as prop and not height and width?

**Review by SavelevMatthew — APPROVED** [review]
Now looks as designed...LGTM

File: apps/condo/domains/common/components/icons/BarBillingIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import Icon from '@ant-design/icons'
+
+interface BarBillingIconProps {
+    viewBox?: string
+}
+
+const BarBillingIconSVG: React.FC<BarBillingIconProps> = ({ viewBox = '0 0 20 20' }) => {
+    return (
+        <svg width="20" height="20" fill="none" viewBox={viewBox} xmlns="http://www.w3.org/2000/svg"><path fillRule="evenodd" clipRule="evenodd" d="M4 2h12c1.1046 0 2 .8954 2 2v5H2V4c0-1.1046.8954-2 2-2Zm-2 9v5c0 1.1046.8954 2 2 2h12c1.1046 0 2-.8954 2-2v-5H2ZM0 4c0-2.2091 1.7909-4 4-4h12c2.2091 0 4 1.7909 4 4v12c0 2.2091-1.7909 4-4 4H4c-2.2091 0-4-1.7909-4-4V4Zm7 2c0-.5523.4477-1 1-1h4c.5523 0 1 .4477 1 1s-.4477 1-1 1H8c-.5523 0-1-.4477-1-1Zm1 7c-.5523 0-1 .4477-1 1s.4477 1 1 1h4c.5523 0 1-.4477 1-1s-.4477-1-1-1H8Z" fill="currentColor"/></svg>
```
Comment (SavelevMatthew) [inline apps/condo/domains/common/components/icons/BarBillingIcon.tsx:10]: Please don't use more than 1 tag in single line....It makes it harder to read

**Pull request merged**